includes/gateways/paypal-standard.php

```diff
@@ -1122,105 +1124,112 @@ function edd_paypal_link_transaction_id( $transaction_id, $payment_id ) {
 add_filter( 'edd_payment_details_transaction_id-paypal', 'edd_paypal_link_transaction_id', 10, 2 );
 
 /**
- * Shows checkbox to automatically refund payments made in PayPal.
+ * Shows a checkbox to automatically refund payments in PayPal.
  *
- * @since  2.6.0
+ * @param Order $order
  *
- * @param int $payment_id The current payment ID.
+ * @since 3.0
  * @return void
  */
-function edd_paypal_refund_admin_js( $payment_id = 0 ) {
-
-	// If not the proper gateway, return early.
-	if ( 'paypal' !== edd_get_payment_gateway( $payment_id ) ) {
+function edd_paypal_refund_checkbox( Order $order ) {
+	if ( 'paypal' !== $order->gateway ) {
 		return;
 	}
 
 	// If our credentials are not set, return early.
-	$key       = edd_get_payment_meta( $payment_id, '_edd_payment_mode', true );
+	$key       = $order->mode;
 	$username  = edd_get_option( 'paypal_' . $key . '_api_username' );
 	$password  = edd_get_option( 'paypal_' . $key . '_api_password' );
 	$signature = edd_get_option( 'paypal_' . $key . '_api_signature' );
 
 	if ( empty( $username ) || empty( $password ) || empty( $signature ) ) {
 		return;
 	}
-
-	// Localize the refund checkbox label.
-	$label = __( 'Refund Payment in PayPal', 'easy-digital-downloads' );
-
 	?>
-	<script type="text/javascript">
-		jQuery(document).ready(function($) {
-			$('select[name=edd-payment-status]').change(function() {
-				if ( 'refunded' === $(this).val() ) {
-					$(this).parent().parent().append('<input type="checkbox" id="edd-paypal-refund" name="edd-paypal-refund" value="1" style="margin-top:0">');
-					$(this).parent().parent().append('<label for="edd-paypal-refund"><?php echo $label; ?></label>');
-				} else {
-					$('#edd-paypal-refund').remove();
-					$('label[for="edd-paypal-refund"]').remove();
-				}
-			});
-		});
-	</script>
+	<div class="edd-form-group edd-paypal-refund-transaction">
+		<div class="edd-form-group__control">
+			<input type="checkbox" id="edd-paypal-refund" name="edd-paypal-refund" value="1">
+			<label for="edd-paypal-refund">
+				<?php esc_html_e( 'Refund Payment in PayPal', 'easy-digital-downloads' ); ?>
```

Comment:
The reason I chose payment is I guess because I see an order as different from a payment. You can have an order without an associated payment attached.

Perhaps `transaction` is actually the word I'm looking for.